packages/react/src/editor/styles.css

```diff
@@ -159,6 +160,12 @@
     color: var(--bn-colors-hovered-text);
 }
 
+/* Mantine Popover component base styles */
+.mantine-Popover-dropdown {
+    all: revert;
```

Comment:
why are these `all: revert` necessary? In general using `all` is not desirable I think (I know I did this somewhere else too, we should probably get rid of that one as well at some point)